CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 - `binary` module created with `from_string`, `byte_size`, `append`, `part`,
   `int_to_u32` and `int_from_u32`.
+- `os` module created with `get_env`, `put_env`, `delete_env`.
```

Comment:
`insert_env` perhaps? We don't use the word `put` anywhere in the stdlib.

Reply:
Just inspired by the erl docs, I am happy to change to insert_env